packages/testing-helpers/src/scopedElementsWrapper.js

```diff
@@ -0,0 +1,64 @@
+import { ScopedElementsMixin } from '@open-wc/scoped-elements';
+import { html, LitElement, TemplateResult } from 'lit-element';
+import { nothing } from 'lit-html';
+import { isIterable } from './lib.js';
+
+const transform = template => {
+  if (isIterable(template)) {
+    return [...template].map(v => transform(v));
+  }
+
+  if (template instanceof TemplateResult) {
+    return html(template.strings, ...template.values);
+  }
+
+  return template;
+};
+
+class ScopedElementsClass extends ScopedElementsMixin(LitElement) {
+  static get properties() {
+    return {
+      scopedElements: { type: Object },
+      template: { type: Object },
+    };
+  }
+
+  constructor() {
+    super();
+
+    this.scopedElements = {};
+    this.template = nothing;
+  }
+
+  async firstUpdated() {
+    // @ts-ignore
+    await super.firstUpdated();
+
+    Object.keys(this.scopedElements).forEach(key =>
+      this.defineScopedElement(key, this.scopedElements[key]),
+    );
+  }
+
+  render() {
+    return transform(this.template);
+  }
+}
+
+// @ts-ignore
+customElements.define('scoped-elements-test-wrapper', ScopedElementsClass);
+
+/**
+ * Wraps the template inside a scopedElements component
+ *
+ * @param {import('./litFixture').LitHTMLRenderable} template
+ * @param {Object<string, typeof HTMLElement>} scopedElements
+ * @returns {import('lit-html').TemplateResult}
+ */
+export function getScopedElementsTemplate(template, scopedElements) {
```

Comment:
does this need to be a function?

Reply:
As it is exported I'm using a function because `tsc` doesn't export the types in case using an arrow function.